builder/pipeline_test.cc

```diff
@@ -1110,6 +1132,9 @@ TEST(pipeline, padded_stencil) {
         ASSERT_EQ(correct, out_buf(x, y)) << x << " " << y;
       }
     }
+
+    // Also visualize this pipeline.
+    visualize(viz_dir() + "padded_stencil_" + std::to_string(schedule) + ".html", p, inputs, outputs, &ctx);
```

Comment:
May I suggest something like `std::cout << "Generating visualization to ${PATH}"` here?

Reply:
This is annoyingly tricky to do in a useful way without more capable path utils that I don't want to bring a dependency in for/rely on `system(...)`. It will basically only ever show "" or the bazel path. But the bazel path is going to get zipped and put somewhere else anyways!